bin/textlint.js

```diff
@@ -4,7 +4,8 @@ var concat = require("concat-stream"),
     cli = require("../lib/cli");
 
 var exitCode = 0,
-    useStdIn = (process.argv.indexOf("--stdin") > -1);
+    useStdIn = (process.argv.indexOf("--stdin") > -1),
+    init = (process.argv.indexOf("--init") > -1);
```

Comment:
Remove this from `bin/`
Instead, Add `init` option to [options.js](https://github.com/textlint/textlint/blob/master/src/options.js "options.js") like [this](https://github.com/eslint/eslint/blob/8ca6bedac12155891d5dca556498ce12a035eccb/lib/options.js#L177-L182).